packages/cli/src/bin/ceramic.ts

```diff
@@ -4,6 +4,7 @@ import { CeramicCliUtils } from '../ceramic-cli-utils'
 
 program
   .command('daemon')
+  .option('--config <path>', 'Path to the Ceramic Daemon config file')
```

Comment:
Is only daemon config file configurable? Would it make more sense to just specify the config directory?

Reply:
this flag is specific to the `daemon` command so there's no reason to take anything other than the path to the daemon config file.  I also don't think it would make sense to make the location of the config file for the cli client be configurable, as you'd have to add the flag to every cli command it doesn't really make sense to try to use the CLI with passing a flag for the config file on every single command you try to run through the cli